jsons/deserializers/default_path.py

```diff
@@ -0,0 +1,13 @@
+from pathlib import PurePath
+
+
+def default_path_deserializer(obj: str, cls: type = PurePath, **kwargs) -> PurePath:
+    """
+    Deserialize a string to a `pathlib.PurePath` object. Since ``pathlib``
+    implements ``PurePath``, no filename or existence checks are performed.
+    :param obj: the string to deserialize.
+    :param kwargs: not used.
+    :return: a ``str``.
+    """
+    path: PurePath = cls(obj)
+    return path
```

Comment:
This will break Python 3.5 compatibility.

```suggestion
    return cls(obj)
```

Strange that the Github Action pipelines didn't activate. 🤔

Reply:
I just did a little bit of investigating, GitHub automatically disables GitHub actions on forked repositories, so I had to explicitly enable it. I have done that now and will make some amends to the PR. Also I noticed that the workflow file only seems to be on "push" rather than on both "push" and "pull_request", so it might be worth enabling it for PRs in a separate feature branch.

Reply:
<img width="1024" alt="Screen Shot 2020-07-05 at 11 10 21 am" src="https://user-images.githubusercontent.com/34053287/86523470-4cc93e00-beb0-11ea-8ac3-89c1afb161a4.png">